### PR TITLE
pre_defined_classes-mathematical_program-medium-q6

### PR DESCRIPTION
**Pull request by Hedgemon4**

### CODE REVIEW

**Review by SanaBob — DISMISSED** [review]
looks good

File: src/pre_defined_classes/mathematical_program/medium/q6/SoundsGoodTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package pre_defined_classes.mathematical_program.medium.q6;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SoundsGoodTest extends BaseTest {
```
Comment (SamamthaH) [inline src/pre_defined_classes/mathematical_program/medium/q6/SoundsGoodTest.java:10]: Change to MainTest and add //Parsons

File: src/pre_defined_classes/mathematical_program/medium/q6/MainTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("A person speaking is "),
+                new DoubleLiteral(160, 170, "decibels"),
```
Comment (SamamthaH) [inline src/pre_defined_classes/mathematical_program/medium/q6/MainTest.java:15]: why the range? From what I see the number shouldn't change

**Review by SamamthaH — APPROVED** [review]

**Review by SamamthaH — APPROVED** [review]

File: src/pre_defined_classes/mathematical_program/medium/q6/SoundsGood.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+intensity of the sound to be measured in W/m2 ( Watts per metre squared ), I0 is the minimum intensity of sound that
+can be heard by the human ear which is known as 10-12 W/m2. Make a Java program that finds the decibel value of a person
+speaking if the intensity of the voice is 10-5 W/m2.
+ */
+
+public class SoundsGood {
+    public static void main(String[] args) {
+        double I0 = Math.pow(10, -12);
+        double voice = Math.pow(10, -5);
+        double Ip = voice;
```
Comment (aadiraju) [inline src/pre_defined_classes/mathematical_program/medium/q6/SoundsGood.java:15]: Looking at this solution, is there even a point to having these be separate variables if we never do anything? Maybe change this to just be one line `double Ip = Math.pow(10,-5);`

Comment (Hedgemon4) [reply src/pre_defined_classes/mathematical_program/medium/q6/SoundsGood.java:15]: Ok sounds good

**Review by Opeyem1a — APPROVED** [review]

**Review by Opeyem1a — APPROVED** [review]

**Pull request merged**